### PR TITLE
default to 3.1.1 in download jar

### PR DESCRIPTION
**Pull request by hzding621**
## Summary

as we are moving away from 2.4.0, remove some references of 2.4.0 and use 3.1.1 as default

## Test Plan

- [ ] Added Unit Tests
- [x] Covered by existing CI
- [ ] Integration tested

## Checklist
- [ ] Documentation update

## Reviewers

@pengyu-hou @yuli-han

### CODE REVIEW

File: api/py/ai/chronon/repo/run.py [hunk 1]
```diff
@@ -196,17 +194,15 @@ def download_jar(
     version,
     jar_type="uber",
     release_tag=None,
-    spark_version="2.4.0",
+    spark_version="3.1.1",
```
Comment (hzding621) [inline api/py/ai/chronon/repo/run.py:197]: this is the main change.

File: api/py/ai/chronon/repo/run.py [hunk 2]
```diff
@@ -610,11 +548,9 @@ def set_defaults(parser):
         repo=chronon_repo_path,
         online_class=os.environ.get("CHRONON_ONLINE_CLASS"),
         version=os.environ.get("VERSION"),
-        spark_version=os.environ.get("SPARK_VERSION", "2.4.0"),
+        spark_version=os.environ.get("SPARK_VERSION", "3.1.1"),
```
Comment (hzding621) [inline api/py/ai/chronon/repo/run.py:551]: this is the main change

**Review by hzding621 — COMMENTED** [review]
there's a lot of lint related changes from git hook

**Review by pengyu-hou — APPROVED** [review]
We are getting there!

**Pull request merged**